Airwallex/Core/Sources/Internal/AWXPaymentMethodListViewController.m

```diff
@@ -140,7 +140,13 @@ - (void)reloadListItems {
 }
 
 - (void)presentSingleCardShortcutIfRequired {
-    BOOL hasPaymentConsents = self.availablePaymentConsents.count > 0 && !self.session.hidePaymentConsents;
+    BOOL hidePaymentConsents = NO;
+    AWXOneOffSession *oneOffSession = (AWXOneOffSession *)self.session;
+    if (oneOffSession) {
+        hidePaymentConsents = oneOffSession.hidePaymentConsents;
+    }
+
+    BOOL hasPaymentConsents = self.availablePaymentConsents.count > 0 && !hidePaymentConsents;
```

Comment:
Do we need to check `!hidePaymentConsents` any more since `availablePaymentConsents.count` is `0` anyway when it's `true`?